### PR TITLE
test scylla with docker

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

**Comment by larisau** [comment]
approved

File: scylla_docker.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+class ScyllaDocker(object):
+    """
+    Implements methods for deploying scylla with docker
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._image = kwargs.get('image', 'scylladb/scylla-nightly')
+        self._node_cnt = kwargs.get('node_cnt', 1)
+        self._seed_name = 'node1'
```
Comment (roydahan) [inline scylla_docker.py]: @larisau , please generate a unique name for each run e.g. node2017...

**Comment by amoskong** [comment]
On Thu, May 11, 2017 at 11:33 PM, Roy Dahan <notifications@github.com>
wrote:

> *@roydahan* commented on this pull request.
> ------------------------------
>
> In scylla_docker.py
> <https://github.com/scylladb/scylla-artifact-tests/pull/54#discussion_r116021711>
> :
>
> > +from avocado import Test
> +from avocado.utils import process
> +from avocado import main
> +
> +log = logging.getLogger('scylla_docker')
> +
> +
> +class ScyllaDocker(object):
> +    """
> +    Implements methods for deploying scylla with docker
> +    """
> +
> +    def __init__(self, *args, **kwargs):
> +        self._image = kwargs.get('image', 'scylladb/scylla-nightly')
> +        self._node_cnt = kwargs.get('node_cnt', 1)
> +        self._seed_name = 'node1'
>
> @larisau <https://github.com/larisau> , please generate a unique name for
> each run e.g. node2017...
>

@larisau strip('1') in create_cluster() will replace all number 1 in the
end of seed_name.
We can generate a name prefix and save to self.node_name_prefix. Use it and
index as the nodes' name.

File: scylla_docker.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+log = logging.getLogger('scylla_docker')
+
+
+class ScyllaDocker(object):
+    """
+    Implements methods for deploying scylla with docker
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._image = kwargs.get('image', 'scylladb/scylla-nightly')
```
Comment (amoskong) [inline scylla_docker.py]: Parameter name in scylla-docker.yaml is `docker_image`, it's not unified.

Comment (roydahan) [inline scylla_docker.py]: ?

Comment (larisau) [inline scylla_docker.py]: it's not parameter from yaml file, it's class parameter

Comment (amoskong) [inline scylla_docker.py]: You are right, ignore my comment.  @larisau

**Pull request merged**